src/tests/ftest/vmd/led.py

```diff
@@ -0,0 +1,161 @@
+"""
+  (C) Copyright 2020-2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import json
+import time
+from osa_utils import OSAUtils
+from avocado.utils import process
+
+
+class VmdLedStatus(OSAUtils):
+    """
+    Test Class Description: This class methods to get
+    the VMD LED status.
+    """
+    # pylint: disable=too-many-instance-attributes,too-many-ancestors
+    def setUp(self):
+        super().setUp()
+        self.targets = self.params.get("targets", "/run/server_config/servers/0/*")
+        self.dmg = self.get_dmg_command()
+        self.dmg.hostlist = self.hostlist_servers[0]
+
+    def get_nvme_device_ids(self):
+        """Get the list of nvme device-ids.
+        Returns: List of uuid
+        """
+        self.dmg.json.value = True
+        try:
+            result = self.dmg.storage_query_list_devices()
+        except process.CmdError as details:
+            self.fail("dmg command failed: {}".format(details))
```

Comment:
You will not see this exception as it has been caught in https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/general_utils.py#L277.  Instead, this will be initially returned as a DaosTestError exception, which is then converted into a CommandFailure.
```suggestion
        except CommandFailure as details:
            self.fail("dmg command failed: {}".format(details))
```

We should also fix `checksum/csum_error_logging.py`, `control/dmg_nvme_scan_test.py` and `deployment/disk_failure.py`.

Reply:
I will other CommandFailure issues in different PR. Don't want to gate this PR with it... Will create a ticket.